app/controllers/general.php

```diff
@@ -941,7 +947,7 @@ function router(App $utopia, Database $dbForConsole, callable $getProjectDB, Swo
 
         $type = $error->getType();
 
-        $output = ((App::isDevelopment())) ? [
+        $output = ((App::isDevelopment()) || (!$devKey->isEmpty())) ? [
```

Comment:
❗ This feels scarry ❗

Please double check with Christy or Eldad, but I think this has potential to expose traces with internal secrets.

Reply:
Yes, here we need a better strategy as to how to filter system details and secrets stuff.

Reply:
This is a blocker for merge.

We had an incident on Cloud with Executor when we had those detailed messages enabled, and community member reported exposure of secret key used for communication with Open Rutimes Executor.